File: Exam-3.January.2018/Skeletons/C#-Skeleton/ProjectRider/Models/Project.cs
using System.ComponentModel.DataAnnotations;

namespace ProjectRider.Models
{
    public class Project
    {
        //TODO: Implement me ...
        /*
        •	id – technology-dependent identifier (ObjectID for JavaScript, int for all other technologies)
        •	title – non-empty text
        •	description – non-empty text
        •	budget – a 64-bit integer
        */

        [Key]
        public int Id { get; set; }

        [Required]
        [MinLength(1)]
        public string Title { get; set; }

        [Required]
        [MinLength(1)]
        public string Description { get; set; }   
   
        public long Budget { get; set; }

    }
}

File: Exam-3.January.2018/Skeletons/JavaScript-Skeleton/models/Project.js
const mongoose = require('mongoose');

let projectSchema = mongoose.Schema({
    //TODO: Implement me ...
    /*
    •	id – technology-dependent identifier (ObjectID for JavaScript, int for all other technologies)
    •	title – non-empty text
    •	description – non-empty text
    •	budget – a 64-bit integer
     */
    title: {type: 'string', required: 'true'},
    description: {type: 'string', required: 'true'},
    budget: {type: 'number', required: 'true'},
});

let Project = mongoose.model('Project', projectSchema);

module.exports = Project;